index.js

```diff
@@ -637,6 +637,23 @@ class ModbusRTU extends EventEmitter {
         return false;
     }
 
+    /**
+     * Clears the timeout for all pending transactions.
+     * This essentially cancels all pending requests.
+     */
+    _cancelPendingTransactions() {
+        if (Object.keys(this._transactions).length > 0) {
+            Object.values(this._transactions).forEach((transaction) => {
+                if (transaction._timeoutHandle) {
+                    _cancelTimeout(transaction._timeoutHandle);
+                }
+            });
+        }
+    }
+
+    // eslint command to fix this file
+    // eslint --fix index.js
```

Comment:
hi, thank you for the pull request, do we need this lines ?

Reply:
Sorry about that. It was a self note to format the code before creating a commit. 

But I guess I forgot to delete it